docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,215 @@
 
 You can use Bluetooth functionalities in your application, such as managing the local Bluetooth adapter, bonding, and exchanging data between Bluetooth-enabled devices. The Bluetooth standard provides a peer-to-peer (P2P) data exchange functionality over short distance between compliant devices.
 
-This feature is supported in mobile and wearable applications only.
+## Main features of Bluetooth API
 
 The main features of the Bluetooth API include:
 
-- Managing the local Bluetooth adapter   
+- Handle UUIDs and binary data
 
-  You can [enable and disable the local Bluetooth adapter](#managing-the-local-bluetooth-adapter), and change the device name for it.
+  You can [convert UUIDs and binary data between equivalent formats and compare UUIDs in different formats](#handle-uuids-and-binary-data).
 
-- Discovering devices   
+- Manage the local Bluetooth adapter
 
-  You can [discover other Bluetooth devices](#discovering-bluetooth-devices).
+  You can [enable and disable the local Bluetooth adapter](#manage-the-local-bluetooth-adapter), and change the device name for it.
 
-- Creating a bond with a Bluetooth device   
+- Discover devices
 
-  You can [create a bond](#creating-a-bond-with-a-bluetooth-device) with another device retrieved through the discovery process. Bonding allows the 2 devices to establish a connection.
+  You can [discover other Bluetooth devices](#discover-bluetooth-devices).
 
-- Connecting to and exchanging data with a Bluetooth device   
+- Create a bond with a Bluetooth device
 
-  You can [connect to and exchange data with a remote Bluetooth device](#connecting-to-and-exchanging-data-with-a-bluetooth-device).
+  You can [create a bond](#create-a-bond-with-a-bluetooth-device) with another device retrieved through the discovery process. Bonding allows the 2 devices to establish a connection.
```

Comment:
Please change 2 devices to two devices:

  You can [create a bond](#create-a-bond-with-a-bluetooth-device) with another device retrieved through the discovery process. Bonding allows the two devices to establish a connection.

Reply:
Done